src/commands/__snapshots__/branches.test.ts.snap

```diff
@@ -263,4 +263,3 @@ created_at: 2019-01-01T00:00:00Z
 updated_at: 2019-01-01T00:00:00Z
 "
 `;
-
```

Comment:
did you manually edit this file? :)

Reply:
I think linter added that change for me, but I couldn't figure out why